flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -470,6 +477,41 @@ public void testCassandraTableSink() throws Exception {
 		Assert.assertTrue("The input data was not completely written to Cassandra", input.isEmpty());
 	}
 
+	@Test
+	public void testCassandraBatchPojoFormat() throws Exception {
+
+		session.execute(CREATE_TABLE_QUERY.replace(TABLE_NAME_VARIABLE, "batches"));
+
+		CassandraPojoSink<CustomCassandraAnnotatedPojo> sink = new CassandraPojoSink<>(CustomCassandraAnnotatedPojo.class, builder);
```

Comment:
I'm a bit torn on this one. I kinda dislike that we're using other sinks for the verification; this test could now fail if the `CassandraPojoSink` is modified in some incompatible way, which simply shouldn't be possible.

Reply:
Let's keep it for now but update it once the outputformat is in.